lib/transbank/sdk/patpass/patpass_by_webpay/errors/patpass_by_webpay_errors.rb

```diff
@@ -0,0 +1,14 @@
+require 'transbank/sdk/patpass/patpass_by_webpay/errors/transaction_create_error'
+require 'transbank/sdk/patpass/patpass_by_webpay/errors/transaction_commit_error'
+require 'transbank/sdk/patpass/patpass_by_webpay/errors/transaction_status_error'
+require 'transbank/sdk/patpass/errors/integration_type_error'
```

Comment:
The directory that the class file is in is inconsistent with its namespace

Reply:
Fixed!